opentelemetry-sdk/src/metrics/instrument.rs

```diff
@@ -185,8 +185,8 @@ impl StreamBuilder {
     /// Set the stream allowed attribute keys.
     ///
     /// Any attribute recorded for the stream with a key not in this set will be
-    /// dropped. If the set is empty, all attributes will be dropped, if `None` all
-    /// attributes will be kept.
+    /// dropped. If the set is empty, all attributes will be dropped.
+    /// If not set, all attributes will be kept.
```

Comment:
I think we can make it more clear without overloading the word "set". Even the line above mentions "set".
```suggestion
    /// If this method is not used, all attributes will be kept.
```